lib/network/contact.js

```diff
@@ -12,6 +14,8 @@ var version = require('../version');
 * @param {String} contact.address - Hostname of IP address
 * @param {Number} contact.port - RPC port number
 * @param {String} contact.nodeID - 160 bit node ID (hex)
+* @param {String} contact.hdKey - extended hd public key
```

Comment:
If these guys are optional, you can wrap them in [] to reflect that in the docs.